app.rb

```diff
@@ -0,0 +1,131 @@
+require './book'
+require './person_class'
+require './student_class'
+require './teacher_class'
+require './rental'
+
+$books = []
+$students = []
+$teachers = []
+$rental = []
+$people = []
+
+def sort_books
+  if $books.empty?
+    puts 'There is no books :('
+  else
+    puts 'Here is the Books :)'
+    $books.each { |b| puts "Title: #{b.title}, Author:#{b.author}" }
+  end
+end
+
+def sort_people
+  if !$students.empty? || !$teachers.empty?
+    $students.each { |s| puts "Student name: #{s.name}, ID: #{s.id}, AGE:#{s.age}" }
+    $teachers.each { |t| puts "Teacher name: #{t.name}, ID: #{t.id}, AGE:#{t.age}" }
+  else
+    puts 'there is no one here :('
+  end
+end
+
+def add_student(classroom, age, name, parent_permission: true)
+  $students << Student.new(classroom, age, name, parent_permission)
+end
+
+def add_teacher(name, age, specialization)
+  $teachers << Teacher.new(name, age, specialization)
+end
+
+def create_person
+  puts 'to create student press 1 , teacher 2 '
+  choice = gets.chomp
+  case choice.to_i
+  when 1
+    print "Enter student's name: "
+    s_name = gets.chomp
+    print 'Enter Age: '
+    s_age = gets.chomp
+    print 'Has parent permission? [y/n]'
+    p_student = gets.chomp
+    case p_student.downcase
+    when 'y'
+      add_student(classroom = 1, age = s_age, name = s_name)
+      puts 'Student created :)'
+    when 'n'
+      add_student(classroom = 1, age = s_age, name = s_name, parent_permission = false)
```

Comment:
- [ ] When I run the app application on the terminal and type 'n' to show that the student has no parent permission, I get an error saying that you gave 4 arguments, expected 3. This happens because you added the 'parent_permission' as a 4th argument. Please fix this.

Reply:
done

Reply:
well resolved! ✔️